### PR TITLE
Only show message if warnings exist.

### PR DESCRIPTION
**Pull request by snide**
The message block currently appears even if no warnings are present. Assume we only want to show messages when there are actual errors.

### CODE REVIEW

**Comment by snide** [comment]
Also went ahead and edited the background property that was appearing malformed in Chrome.

image

File: index.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = postcss.plugin('postcss-messages', function (opts) {
...
+        '#DF4F5E url("data:image/svg+xml;charset=utf-8,%3Csvg%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20x%3D%220px%22%20y%3D%220px%22%20width%3D%2248px%22%20height%3D%2248px%22%20viewBox%3D%220%200%20512%20512%22%20enable-background%3D%22new%200%200%20512%20512%22%20xml%3Aspace%3D%22preserve%22%3E%3Cpath%20fill%3D%22%23A82734%22%20id%3D%22warning-4-icon%22%20d%3D%22M228.55%2C134.812h54.9v166.5h-54.9V134.812z%20M256%2C385.188c-16.362%2C0-29.626-13.264-29.626-29.625c0-16.362%2C13.264-29.627%2C29.626-29.627c16.361%2C0%2C29.625%2C13.265%2C29.625%2C29.627C285.625%2C371.924%2C272.361%2C385.188%2C256%2C385.188z%20M256%2C90c91.742%2C0%2C166%2C74.245%2C166%2C166c0%2C91.741-74.245%2C166-166%2C166c-91.742%2C0-166-74.245-166-166C90%2C164.259%2C164.245%2C90%2C256%2C90z%20M256%2C50C142.229%2C50%2C50%2C142.229%2C50%2C256s92.229%2C206%2C206%2C206s206-92.229%2C206-206S369.771%2C50%2C256%2C50z%22%2F%3E%3C%2Fsvg%3E") .5em no-repeat)',
```
Comment (MoOx) [inline index.js:24]: You just remove the background gradient. It's not how this should be fixed.

**Comment by MoOx** [comment]
Weird that nobody get those issues before.

File: index.js (hunk 1 above)
Comment (snide) [inline index.js:24]: Seemed overly stylized for a simple thing imho.

Comment (MoOx) [inline index.js:24]: In this case provide a screenshot an be explicit about the change ;)

Comment (snide) [inline index.js:24]: Didn't mean that comment to sound harsh. I know you can restyle them, which is what I've done, just thought a simpler style might be better.

**Comment by snide** [comment]
Sorry for the quick PR, still learning a bit about postcss, it's awesome! I submitted a fix for the tests, and removed the linear gradient and just made it a straight background color, which is a simpler render and less prone to browser differences. Here's a screen of updated look.

image

**Comment by gazay** [comment]
Thank you for PR, I like #13 more as it doesn't change styles and return right in beginning if there no messages

**Pull request closed without merging**